backend/LeetWars.Core/LeetWars.Core.BLL/Services/UserService.cs

```diff
@@ -0,0 +1,30 @@
+using AutoMapper;
+using LeetWars.Core.Common.DTO.User;
+using LeetWars.Core.DAL.Context;
+using LeetWars.Core.DAL.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace LeetWars.Core.BLL.Services;
+
+public class UserService : BaseService, IUserService
+{
+    public UserService(LeetWarsCoreContext context, IMapper mapper) : base(context, mapper)
+    {
+    }
+
+    public async Task<UserDto> CreateUserAsync(NewUserDto userDto)
+    {
+        if (userDto is null)

```

Comment:
Violation B5 of  [code conventions](https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/dotnet.md)

Reply:
Can you explain how I break B5 convention? I don’t understand